src/lib/polymer-bootstrap.html

```diff
@@ -37,13 +37,28 @@
       var options = {
         prototype: prototype
       };
-      // NOTE: we're specifically supporting older Chrome versions here 
+      // NOTE: we're specifically supporting older Chrome versions here
       // (specifically Chrome 39) that throw when options.extends is undefined.
       if (prototype.extends) {
         options.extends = prototype.extends;
       }
-      Polymer.telemetry._registrate(prototype);
-      document.registerElement(prototype.is, options);
+
+      try {
```

Comment:
The only other option is to try creating the element and see if it has a prototype that isn't `HTMLElement`

```js
var e = prototype.extends, i = prototype.is;
var el = e ? document.createElement(e, i) : document.createElement(e);
if (Object.getPrototype(el) !== HTMLElement.prototype) {
  // ...
}
```